File: dotnet_winform_simpleLotto/simpleLotto/model/LottoDrawResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace simpleLotto.model {
    public record class LottoDrawResult( 
        String returnValue = null, // 요청결과
        String drwNoDate = null,// 날짜
        long totSellamnt = 0,
        long firstWinamnt = 0, //1등 상금액
        int firstPrzwnerCo = 0, // 1등 당첨인원
        long firstAccumamnt = 0,
        int drwtNo1 = 0,// 로또번호 1
        int drwtNo2 = 0,// 로또번호 2
        int drwtNo3 = 0,// 로또번호 3
        int drwtNo4 = 0,// 로또번호 4
        int drwtNo5 = 0,// 로또번호 5
        int drwtNo6 = 0,// 로또번호 6
        int bnusNo = 0,// 로또 보너스 번호
        int drwNo = 0 // 로또회차
   );
}

File: streamlit_lottoResult/app/data/model/lotto_draw_result.py
class LottoDrawResult:
    drwNo = int
    drwNoDate = str
    drwtNo1 = int
    drwtNo2 = int
    drwtNo3 = int
    drwtNo4 = int
    drwtNo5 = int
    drwtNo6 = int
    bnusNo = int
    totSellamnt = int
    firstWinamnt = int
    firstPrzwnerCo = int
    firstAccumamnt = int
    returnValue = str

    def __init__(
        self,
        drwNo ,
        drwNoDate ,
        drwtNo1 ,
        drwtNo2 ,
        drwtNo3 ,
        drwtNo4 ,
        drwtNo5 ,
        drwtNo6 ,
        bnusNo ,
        totSellamnt ,
        firstWinamnt ,
        firstPrzwnerCo ,
        firstAccumamnt ,
        returnValue):
        self.drwNo = drwNo
        self.drwNoDate = drwNoDate
        self.drwtNo1 = drwtNo1
        self.drwtNo2 = drwtNo2
        self.drwtNo3 = drwtNo3
        self.drwtNo4 = drwtNo4
        self.drwtNo5 = drwtNo5
        self.drwtNo6 = drwtNo6
        self.bnusNo = bnusNo
        self.totSellamnt = totSellamnt
        self.firstWinamnt = firstWinamnt
        self.firstPrzwnerCo = firstPrzwnerCo
        self.firstAccumamnt = firstAccumamnt
        self.returnValue = returnValue